server/server.go

```diff
@@ -89,12 +89,16 @@ func RootHandler(ac auth.AccessController, ctx context.Context, trust signed.Cry
 		prometheus.InstrumentHandlerWithOpts(
 			prometheusOpts("UpdateTuf"),
 			hand(handlers.AtomicUpdateHandler, "push", "pull")))
-	r.Methods("GET").Path("/v2/{imageName:.*}/_trust/tuf/{tufRole:(root|targets(?:/[^/\\s]+)*|snapshot|timestamp)}.json").Handler(
```

Comment:
They were actually incorrect. It's not explicitly stated in the mux docs anywhere but after debugging my URL regex problems yesterday and getting help in IRC, we determined that not only are they not needed, they shouldn't be used. It causes unexpected behaviour.